hbase-protocol-shaded/src/main/protobuf/Client.proto

```diff
@@ -511,12 +513,15 @@ message RegionActionResult {
 message MultiRequest {
   repeated RegionAction regionAction = 1;
   optional uint64 nonceGroup = 2;
+  // Moved this to RegionAction in HBASE-8458. Keep it for backward compatibility. Need to remove
+  // it in the future.
```

Comment:
See "deprecated (field option):" in https://developers.google.com/protocol-buffers/docs/proto ... Seems appropriate here.